source/documentation/upgrade_guide/chap-Red_Hat_Virtualization_Upgrade_Overview.adoc

```diff
@@ -46,9 +46,18 @@ Select the appropriate instructions for your environment from the following tabl
 * Manual upgrade: xref:SHE_Upgrading_from_4-2[]
 
 
-|4.3.x |4.3.y a|xref:Minor_Release_Updates[]
+|4.3 |4.4 a| *Local database environment:*
+
+* Manual upgrade: xref:Upgrading_from_4-3[]
+
+*Remote database environment:* xref:Remote_Upgrading_from_4-3[]
+
+*Self-hosted engine, local database environment:*
+
+* Manual upgrade: xref:SHE_Upgrading_from_4-3[]
+
 |===
 
 ifdef::rhv-doc[]
-For interactive upgrade instructions, you can also use the RHV Upgrade Helper available at https://access.redhat.com/labs/rhvupgradehelper/. This application asks you to provide information about your upgrade path and your current environment, and presents the relevant steps for upgrade as well as steps to prevent known issues specific to your upgrade scenario.
+For interactive upgrade instructions, you can also use the RHV Upgrade Helper available at https://access.redhat.com/labs/rhvupgradehelper/. This application asks you to provide information about your upgrade path and your current environment, and presents the relevant steps for upgrade, as well as steps to prevent known issues specific to your upgrade scenario.
```

Comment:
```suggestion
For interactive upgrade instructions, you can also use the link:https://access.redhat.com/labs/rhvupgradehelper/[RHV Upgrade Helper]. This application asks you to provide information about your upgrade path and your current environment, and presents the applicable upgrade steps, as well as steps to prevent known issues specific to your upgrade scenario.
```